includes/services.inc.php

```diff
@@ -129,7 +129,7 @@ function poll_service($service)
 
     // If we do not have a cmd from the check script, build one.
     if ($check_cmd == "") {
-        $check_cmd = $config['nagios_plugins'] . "/check_" . $service['service_type'] . " -H " . ($service['service_ip'] ? $service['service_ip'] : $service['hostname']);
```

Comment:
Ok! I got it. It works fine. Thanks a lot! 
But any way it is better to add SysName near the Hostname. Please do it.
Also my question how to add SysName to Alert Widget window?
![sysname](https://user-images.githubusercontent.com/17820730/27458113-b5dde802-5764-11e7-98bb-b961a3adbc2b.png)
Because it is too hard to understand what is host in huge list